packages/nodes-base/nodes/Google/Gmail/v1/DraftDescription.ts

```diff
@@ -159,6 +159,14 @@ export const draftFields: INodeProperties[] = [
 				placeholder: 'info@example.com',
 				default: [],
 			},
+			{
+				displayName: 'Reply To Email',
+				name: 'replyTo',
+				type: 'string',
+				placeholder: 'reply@example.com',
+				default: '',
+				description: 'The email address that the reply message is sent to',
+			},
```

Comment:
v1 exist only for compatibility, please do not modify it, unless it is a bug fix